ubuntu22.04/Dockerfile

```diff
@@ -93,6 +93,13 @@ WORKDIR  /drivers
 ARG PUBLIC_KEY=empty
 COPY ${PUBLIC_KEY} kernel/pubkey.x509
 
+# Install the gcc-12 package in Ubuntu 22.04 as Kernels with versions 5.19.x and 6.5.x need gcc 12.3.0 for compilation
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends gcc-12 g++-12 && \
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 12 && \
+    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 12
+
```

Comment:
Can we clean up the `apt` cache here to reduce the layer size:
```
rm -rf /var/lib/apt/lists/*
```

Reply:
oh yes, good catch. Thanks!

Reply:
Done, please check now